### PR TITLE
feat(PushPromotions): allow promotion name to be empty, also return pull request URL

### PR DESCRIPTION
**Pull request by mrparkers**

### CODE REVIEW

File: internal/releaserserver/server.go [hunk 1]
```diff
@@ -186,6 +197,18 @@ func (s *Server) commitAndPushPromotion(ctx context.Context, applicationName, re
...
 }
 
+func (s *Server) getPullRequestURL(prNumber int64) string {
+	return fmt.Sprintf("%s/pull/%d", s.Repo.URL, prNumber)
+}
+
+func getMD5HashForApplications(applications []string) string {
+	hash := md5.New()
+	for _, application := range applications {
+		hash.Write([]byte(application))
```
Comment (cep21) [inline internal/releaserserver/server.go:207]: tisk tisk no error checking!

Comment (cep21) [reply internal/releaserserver/server.go:207]: I'm surprised it passes lint

**Review by cep21 — APPROVED** [review]

File: internal/releaserserver/server.go (hunk 1 above)
Comment (mrparkers) [reply internal/releaserserver/server.go:207]: lol copilot wrote this

Comment (mrparkers) [reply internal/releaserserver/server.go]: not only does it pass lint, but goland doesn't even highlight this as an unhandled error

**Pull request merged**